smoke-tests-otel-starter/spring-boot-3/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -11,4 +11,538 @@
     ],
     "name": "org.apache.commons.pool2.impl.DefaultEvictionPolicy"
   }
+,
```

Comment:
Because of a Gradle bug, we have to disable the access to the graalvm-reachability-metadata repository: https://github.com/graalvm/native-build-tools/issues/572

We have to report the Hikari reflection hints for the new test: https://github.com/oracle/graalvm-reachability-metadata/blob/master/metadata/com.zaxxer/HikariCP/5.0.1/reflect-config.json